test/end2end.py

```diff
@@ -64,8 +65,10 @@ def setUp(self):
         # add /bin to $PATH, /lib to $PYTHONPATH
         self.topdir = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
         self.mympiscript = os.path.join(os.path.join(self.topdir, 'bin'), 'mympirun.py')
-        os.environ['PYTHONPATH'] = '%s:%s' % (os.path.join(self.topdir, 'lib'), os.getenv('PYTHONPATH', ''))
-
+        lib = os.path.join(self.topdir, 'lib')
+        pyver = '%s.%s' % (sys.version_info.major, sys.version_info.minor)
+        eggs = ':'.join(glob.glob(os.path.join(self.topdir, '.eggs', '*.egg')))
+        os.environ['PYTHONPATH'] = '%s:%s:%s' % (eggs, lib, os.getenv('PYTHONPATH', ''))
```

Comment:
please print the value of `$PYTHONPATH` for debugging (although it looks correct)